pyqtgraph/graphicsItems/GradientEditorItem.py

```diff
@@ -25,6 +24,14 @@
     ('grey', {'ticks': [(0.0, (0, 0, 0, 255)), (1.0, (255, 255, 255, 255))], 'mode': 'rgb'}),
 ])
 
+def addGradientListToDocstring():
+    ### create a decorator so that we can add construct a list of the gradients defined above in a docstring
+    ### Adds the list of gradients to the end of the functions docstring
```

Comment:
Can you rewrite this in a more standard docstring format with 
```
"""
Decorator to add a list of valid gradients to the function docstring
"""
```

Feel free to write whatever prose you want between the triple quotes, the above is merely a suggestion.

I like the very verbose function name that clearly defines what it is that this function does :smile: